### ISSUE TITLE
oss-cad-suite can not be downloaded when looking for 'linux'

### ISSUE THREAD

**Issue by waterwin** [FPGAwars/apio#306]
apio install oss-cad-suite
/usr/local/lib/python3.9/dist-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.5) or chardet (5.1.0)/charset_normalizer (2.0.12) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported "
(DEBUG) Profile path: /home/erwin/.apio/profile.json
(DEBUG) Home_dir: /home/erwin/.apio
Installing oss-cad-suite package:
Warning: full platform does not match: linux_i686                
         Trying OS name: linux                
Error: Got an unrecognized status code '404' when downloaded https://github.com/FPGAwars/tools-oss-cad-suite/releases/download/v0.0.8/tools-oss-cad-suite-linux-0.0.8.tar.gz

**Comment by Obijuan** [comment]
It seem that there are some depencencies missing in your system. Could you please try with the latest apio development version? (apio 0.8.2). You can install it with this command:

* **Linux** and **Mac**:

```
sudo pip3 install -U git+https://github.com/FPGAwars/apio.git@develop#egg=apio
```
* **Windows**:
```
pip3 install -U git+https://github.com/FPGAwars/apio.git@develop#egg=apio
```

**Comment by waterwin** [comment]
As far as I can tell no change from previous situation.
***@***.***:~/Documents/apio$ lsapio.ini  icestick  leds.pcf  leds_tb.gtkw  leds_tb.v  ***@***.***:~/Documents/apio$ apio verify(DEBUG) Profile path: /home/erwin/.apio/profile.json(DEBUG) Home_dir: /home/erwin/.apioError: package 'oss-cad-suite' is not installedPlease run:   apio install oss-cad-suite


Installed on a clean Ubuntu VM (Oracle VirtualBox) that has as extra only pip and git installed

Erwin

    On Monday, February 6, 2023 at 12:41:18 AM GMT+1, Juan Gonzalez-Gomez ***@***.***> wrote:  
 
 


It seem that there are some depencencies missing in your system. Could you please try with the latest apio development version? (apio 0.8.2). You can install it with this command:
   
   - Linux and Mac:
sudo pip3 install -U ***@***.***#egg=apio
   
   - Windows:
pip3 install -U ***@***.***#egg=apio

**Comment by Obijuan** [comment]
A new apio has been released (0.8.2). Could you please try it again?

**Comment by Obijuan** [comment]
As there were no answer from the previous question, I think that is has been solved
Please, reopen it if the problem remains for you
